faux-mgs/src/main.rs

```diff
@@ -373,6 +385,13 @@ enum Command {
 
     /// Reads the lock status of any VPD in the system
     VpdLockStatus,
+
+    /// Read the RoT's boot-time information.
+    RotBootInfo {
+        /// Return highest version of RotBootInfo less then or equal to given
+        #[clap(long, short, default_value = "3")]
```

Comment:
I don't love the `"3"` here because I'm worried it'll be easy to forget to update; if we add a new boot info version, `3` will still be valid but won't be the default we'd actually want, right?

I'm not sure how best to fix this. Maybe we could add a `RotBootInfo::latest_version()` function and use that as the default value here? It has the same problem in that it would just return `3` today, but I think it'd be much less likely to be overlooked if someone is adding a variant to `RotBotInfo`.

Reply:
Since versions could be skewed, we want greatest common understood version. So not really "latest_version" but more like RotBootInfo::limit_highest_common_version() or something.

Reply:
I'm not sure I follow that. How would faux-mgs know what the greatest common understood version is?

Reply:
It knows what it knows, it doesn't know what the RoT knows. But, once faux-mgs tells the RoT: "Give me the highest version you  know not greater than this version" the greatest common version is known.

Reply:
Based on https://github.com/oxidecomputer/management-gateway-service/pull/209#discussion_r1575316155, shouldn't this always be "the highest version faux-mgs knows about"? Which would make `latest_version()` make sense?

Reply:
I added a pub const next to the RotBootInfo definition and now have faux-mgs using that constant.